template/web/app/(demo)/mint/_contracts/useCustom1155Contract.test.ts

```diff
@@ -0,0 +1,15 @@
+import { baseSepolia } from 'viem/chains';
+import Custom1155ABI from './Custom1155ABI';
+import { useCustom1155Contract } from './useCustom1155Contract';
+
```

Comment:
Let's avoid this pattern `template/web/app/(demo)` where there is a `demo` folder.

Reply:
each page that comes with BOAT is already a full functioning demo experience.